src/CosmosDB/CosmosDB.Test/ScenarioTests/RestoreTests.ps1

```diff
@@ -784,4 +784,54 @@ function Test-ProvisionCosmosDBAccountBackupPolicyWithContinuous7DaysCmdLets {
   Assert-NotNull $sourceRestorableAccount.DatabaseAccountName
   Assert-NotNull $sourceRestorableAccount.CreationTime
   Assert-NotNull $sourceRestorableAccount.OldestRestorableTime
+}
+
+function Test-CrossRegionRestoreAccountCmdlets {
+  #use an existing account with the following information
+  $rgName = "PSCosmosDBResourceGroup53"
+  $cosmosDBAccountName = "restored-ps-xrr-cosmosdb-12105"
```

Comment:
i think it's better to call it ps-xrr-cosmosdb-12105-restored

Reply:
updated